playbooks/common/openshift-master/config.yml

```diff
@@ -69,6 +73,11 @@
       openshift_env:
         openshift_hosted_registry_storage_kind: 'nfs'
     when: openshift_hosted_registry_storage_kind is not defined and groups.oo_nfs_to_config is defined and groups.oo_nfs_to_config | length > 0
```

Comment:
@abutcher that works, I'm assuming you are running those steps using the oo_masters_to_config group and doing a limit of 1 then?